Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml

```diff
@@ -0,0 +1,325 @@
+﻿- type: entity
+  parent: BaseStructure
+  id: FenceMetalBase
+  name: chain link fence
+  description: A metal piece of fencing cordoning off something likely very important.
+  abstract: true
+  components:
+  - type: MeleeSound
+    soundGroups:
+      Brute:
+        path:
+          "/Audio/Weapons/grille_hit.ogg"
+  - type: Tag
+    tags:
+    - RCDDeconstructWhitelist
+  - type: Sprite
+    sprite: Structures/Walls/fence.rsi
+    drawdepth: WallTops
+  - type: Physics
+    bodyType: Static
+  - type: Damageable
```

Comment:
Should it be anchored too?

Reply:
yes oops

Reply:
actually basestructure is already anchored